rocky/templates/forms/widgets/checkbox_group_table.html

```diff
@@ -7,7 +7,8 @@
       <thead>
         <tr>
           <th>
-            {% if widget.options|length > 1 %}<input name="toggle-all" type="checkbox"/>{% endif %}
+            {# TODO: data-toggle-target is not model-agnostic like the rest of the table #}
+            {% if widget.options|length > 1 %}<input class="toggle-all" data-toggle-target="ooi" type="checkbox"/>{% endif %}
```

Comment:
Inputs should not contain a trailing / anymore. html5 tags are self closing.

Reply:
This is not consistent yet in the codebase as far as I can see. I will update it here but this probably requires a separate ticket as well.

Reply:
https://github.com/minvws/nl-kat-rocky/issues/97